cukas.js

```diff
@@ -1,22 +1,53 @@
+class CukasExceptions {
+    constructor(type ,message) {
+        this.type = type;
+        this.message = message;
+    }
+    toString() {
+        return `Kļūda: ${this.type}, ${this.message}`;
+    }
+}
```

Comment:
maybe should be moved to a separate file?

Reply:
probably yes, it is more like a util

Reply:
Ja tur jau ir CukasGame un CukasPlayer, tad kāpēc CukasGameException būtu pelnījis dzīvot citur? Nav jau milzīgas megaklases.